lamindb/_curate.py

```diff
@@ -265,28 +277,87 @@ def add_new_from(self, key: str, organism: str | None = None, **kwargs):
         Args:
             key: The key referencing the slot in the DataFrame from which to draw terms.
             organism: The organism name.
-            **kwargs: Additional keyword arguments to pass to the registry model.
+            **kwargs: Additional keyword arguments to pass to create new records
```

Comment:
What do you suggest? It's needed for populating things like `add_new_from("perturbation", system="CRISPR-Cas9")`, otherwise it will error as system is required in `wl.GeneticPerturbation`.

Reply:
Hmm they're more or less just for developers and not for users. So we could remove the docstring or label them for internal use only or just leave it as you have it.

P.S. I find it funny that you intuitively used `GeneticPerturbation` when it's called `GeneticTreatment`. Maybe we should just rename them...